src/api/app/policies/attrib_policy.rb

```diff
@@ -39,4 +20,32 @@ def update?
   def destroy?
     create?
   end
+
+  private
+
+  # Parses AttribTypeModifiableBy and AttribNamespaceModifiableBy association
+  # collections and checks the users permissions.
+  #
+  # Permission check is based on:
+  #   - User
+  #   - Group association of user
+  #   - Role association of user (Only for AttribTypeModifiableBy)
+  def has_permissions_for_modifiables?(attrib_modifiable_bies)
```

Comment:
ahh i see its meant to be the plural of `AttribTypeModifiableBy`... Kind of a tricky situation, better than what I thought before that it was just some made up word lol!